### PR TITLE
Encryption support

### PR DESCRIPTION
**Pull request by renato-farias**

### CODE REVIEW

**Comment by renato-farias** [comment]
Hi @guillaumeblaquiere , could you please review this PR. I am adding an encryption support to this library.

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 setup(name='berglas_python',
-      version='0.2.6',
+      version='0.3.0',
       url='https://github.com/guillaumeblaquiere/berglas-python',
       license='Apache 2.0',
       author='Guillaume Blaquiere',
```
Comment (guillaumeblaquiere) [inline setup.py:7]: Add your name as author if you want.

File: berglas_python.py [hunk 2]
```diff
@@ -100,8 +100,7 @@ def Resolve(project_id: str, env_var_value: str) -> str:
     """
 
     if not env_var_value.startswith(BERGLAS_PREFIX):
-        logging.info(f"No berglas prefix for the env var value {env_var_value}")
-        return
+        env_var_value = f'{BERGLAS_PREFIX}{env_var_value}'
```
Comment (guillaumeblaquiere) [inline berglas_python.py]: No, the decryption process will failed if the EnvVar not start by berglas://
User can chose to pass all its env var to this lib and, only the eligible ones are decrypted, the other are return as is.

**Review by guillaumeblaquiere — CHANGES_REQUESTED** [review]
Thanks for the work, refactoring and your experience on Crypto lib in Python. Only the test on the EnvVar name is to change.

File: berglas_python.py (hunk 2 above)
Comment (renato-farias) [reply berglas_python.py]: I corrected this [in this commit](https://github.com/guillaumeblaquiere/berglas-python/pull/1/commits/c2f4165c934b23f71772648dc4bb1b8691d61904)

File: setup.py (hunk 1 above)
Comment (renato-farias) [reply setup.py:7]: I really don't mind @guillaumeblaquiere

**Review by guillaumeblaquiere — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**